neo/Ledger/ContractState.cs

```diff
@@ -72,5 +74,18 @@ public static ContractState FromJson(JObject json)
             contractState.Manifest = ContractManifest.FromJson(json["manifest"]);
             return contractState;
         }
+
+        public StackItem ToStackItem()
+        {
+            return new VM.Types.Array
+           (
+               new StackItem[]
```

Comment:
This is a very good place to apply the proposed ABI serialization I've been discussing... if we adopt such byte serialization format, its conversion to interop could be done automatically like this.